src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,105 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidUserDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int VALID_AGE = 22;
+    private static final String VALID_PASSWORD = "123456";
+    private static final String VALID_LOGIN = "nazari";
+    private static final int INVALID_AGE = 17;
+    private static final String INVALID_LOGIN = "nazar";
+    private static final String INVALID_PASSWORD = "12345";
+    private static RegistrationServiceImpl registrationService;
+    private static User corectUser;
+
+    @BeforeAll
+     static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+     void createUser() {
+        corectUser = new User(VALID_LOGIN,VALID_PASSWORD,VALID_AGE);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    public void register_validUser_OK() {
+        registrationService.register(corectUser);
+        assertTrue(Storage.people.contains(corectUser),"Does not add the user");
+    }
+
+    @Test
+    public void register_NullAge_notOk() {
```

Comment:
```suggestion
    public void register_nullAge_notOk() {
```
please fix similar issues in all places